### DISCUSSION TITLE
В итоге, зачем тебе Искусственный Интеллект?

### DISCUSSION THREAD

**Discussion by katikn** [D1ffic00lt/DPCoinBot#69] [Q&A] [answered]
На планете Земля в настоящее время в нашем мире в 21 веке 23 года 20 февраля в России в городе Новосибирск мы изучаем Искусственный Интеллект, но зачем? В мире столько разных профессий, например слесарь, дровосек, алкаш, но мы почему-то выбираем Искусственный Интеллект.. И я всё-таки хочу понять зачем нам Искусственный Интеллект, ведь можно изучать биологию, химию и физику на уроке Ирины Цербер Викторовны, и радоваться жизни, а потом уехать в Барнаул, но мы почему-то предпочитаем Искусственный Интеллект, почему? Жду ваши ответики, dear followers и дификульт <3 <3 <3

**Comment by D1ffic00lt** [comment, marked as answer]
чтобы генерировать рейтинг (/lb Репутация)

**Reply by katikn** [reply]
ПАСИБО ДИФИКИУЛТЬ UR SO COOOOOOL THANKS <3 <3 <3 <3 <3 <3 <3 <3 <3 <3 <3 <3 <3 <3 <3 <3
